wp-content/plugins/core/src/Blocks/Types/Card_Grid/Card_Grid.php

```diff
@@ -29,7 +29,9 @@ class Card_Grid extends Block_Config {
 	public const LAYOUT_INLINE    = 'inline';
 
 	/**
-	 * Register the block
+	 * 	 * Register the block
```

Comment:
Not sure what's up with spacing on these comments or if I did that in another PR by accident?